packages/svelte/src/reactivity/media-query.js

```diff
@@ -0,0 +1,35 @@
+import { get } from '../internal/client/runtime.js';
+import { set, source } from '../internal/client/reactivity/sources.js';
+import { effect_tracking } from '../internal/client/reactivity/effects.js';
+import { createStartStopNotifier } from './start-stop-notifier.js';
+
+/**
+ * Creates a media query and provides a `current` property that reflects whether or not it matches.
+ */
+export class MediaQuery {
+	#version = source(0);
+	#query;
+	#notify;
+
+	get current() {
+		if (effect_tracking()) {
+			get(this.#version);
+			this.#notify();
+		}
+
+		return this.#query.matches;
+	}
+
+	/**
+	 * @param {string} query A media query string (don't forget the braces)
```

Comment:
Additional parens (as in `((min-width: 800px))`) are ignored, so we can add surrounding parens ourselves and make this unnecessary

```suggestion
	 * @param {string} query A media query string
```

Reply:
Reference https://github.com/sveltejs/svelte/issues/14695